flow/designs/sky130hd/chameleon_hier/DMC_32x16HC/config.mk

```diff
@@ -0,0 +1,44 @@
+
+undefine BLOCKS
+undefine ADDITIONAL_LEFS
+undefine ADDITIONAL_GDS 
+undefine DONT_USE_SC_LIB
+undefine RCX_RULES
```

Comment:
Why `undefine`s? that seems pretty non-standard.

Reply:
This is needed because these variables are already defined at the Top-level and will contradict with macro-level implementation.
The undefines are being done in macro-level config only.

Reply:
I feel like there has to be a better way to do this, but I don't really have the cycles to look at it.